src/py/flwr/cli/new/templates/app/pyproject.sklearn.toml.tpl

```diff
@@ -18,3 +18,7 @@ dependencies = [
 
 [tool.hatch.build.targets.wheel]
 packages = ["."]
+
+[flower.components]
+serverapp = "$module_name.server:app"
+clientapp = "$module_name.client:app"
```

Comment:
```suggestion
serverapp = "$import_name.server:app"
clientapp = "$import_name.client:app"
```

Reply:
@danieljanes In this case we actually want the module name, i.e, the same name as the folder containing the Python files, not the import name (with dashes replaced with underscores)

Reply:
From tests I made, this corresponds to the `package name` (dots, underscores and capital letters allowed)